src/hotspot/share/gc/z/zGlobals.hpp

```diff
@@ -43,11 +43,13 @@ const size_t      ZMaxVirtualReservations       = 100; // Each reservation at le
 
 // Page size shifts
 const size_t      ZPageSizeSmallShift           = ZGranuleSizeShift;
-extern size_t     ZPageSizeMediumShift;
+extern int        ZPageSizeMediumShift;
```

Comment:
Changing to int seems good, but this leaves an inconsistency with `ZPageSizeSmallShift`. I'd prefer if the type of that constant was also changed in this PR.

Reply:
Yeah, was unsure how far to push this. Eventually all our shift variables should probably be typed as int. But will change the page size shifts at least then.